iconservice/iconscore/icon_pre_validator.py

```diff
@@ -31,6 +32,23 @@
 
 
 TAG = "PV"
+REQUEST_PARAMS = (
+    ConstantKeys.VERSION,
+    ConstantKeys.STEP_LIMIT,
+    ConstantKeys.NID,
+    ConstantKeys.TIMESTAMP,
+    ConstantKeys.VALUE,
+    ConstantKeys.NONCE,
+    ConstantKeys.FROM,
+    ConstantKeys.TO,
+    ConstantKeys.SIGNATURE,
+    ConstantKeys.DATA_TYPE,
+    ConstantKeys.DATA,
+)
+
+REQUIRED_PARAMS = REQUEST_PARAMS[:4] + REQUEST_PARAMS[6:9]
+INT_PARAMS = REQUEST_PARAMS[:6]
+ADDR_PARAMS = REQUEST_PARAMS[6:8]
```

Comment:
Another way to improve code readability?

Reply:
What about change code like this?
```
REQUIRED_PARAMS = (
    ConstantKeys.VERSION,
    ConstantKeys.STEP_LIMIT,
    ConstantKeys.NID,
    ConstantKeys.TIMESTAMP,
    ConstantKeys.FROM,
    ConstantKeys.TO,
    ConstantKeys.SIGNATURE
)
INT_PARAMS = (
    ConstantKeys.VERSION,
    ConstantKeys.STEP_LIMIT,
    ConstantKeys.NID,
    ConstantKeys.TIMESTAMP,
    ConstantKeys.VALUE,
    ConstantKeys.NONCE
)
ADDR_PARAMS = (
    ConstantKeys.FROM,
    ConstantKeys.TO
)
```